frontend/packages/dev-console/integration-tests/features/Knative/actions-on-knative-service.feature

```diff
@@ -0,0 +1,212 @@
+Feature: Perform actions on knative service
+   As a user I want to perform edit or delete operations and Set Traffic Distribution on knative Service in topology page
+
+Background:
+   Given user has installed OpenShift Serverless Operator
+   And user is at developer perspective
+   And user has selected namespace "aut-create-knative-actions-service"
+   And user has created knative service "nodejs-ex-git"
+
+
+@regression-1
```

Comment:
@gajanan-more - I added -1 for debugging purpose, Can you please them from all the files

Reply:
Updated.